integration-tests/gradle/src/test/resources/test-uber-jar-format-works/settings.gradle

```diff
@@ -0,0 +1,11 @@
+pluginManagement {
+    repositories {
+        mavenLocal()
+        mavenCentral()
+        gradlePluginPortal()
+    }
+    plugins {
+      id 'io.quarkus' version "${quarkusPluginVersion}"
+    }
+}
+rootProject.name='code-with-quarkus'
```

Comment:
Let's give it a more descriptive name, like `uber-jar-test`

Reply:
Cool